### PR TITLE
Uml 1115 admin app activation keys

### PR DESCRIPTION
**Pull request by lukejolliffe**
# Purpose

Allow users to search for Activation keys that have been saved to our database when added

Fixes UML-1115

## Checklist

* [x] I have performed a self-review of my own code
* [x] I have added relevant logging with appropriate levels to my code
* [] I have added tests to prove my work (Tests will be added in future tickets since none are really feasible without large refactors)
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-api/docker/seeding/start.sh [hunk 1]
```diff
@@ -57,13 +57,14 @@ if ! [[ -z "${AWS_ENDPOINT_DYNAMODB}" ]]; then
 
 
     aws dynamodb create-table \
-    --attribute-definitions AttributeName=Id,AttributeType=S AttributeName=UserId,AttributeType=S \
+    --attribute-definitions AttributeName=Id,AttributeType=S AttributeName=UserId,AttributeType=S AttributeName=ActivationCode,AttributeType=S \
```
Comment (cooperaj) [inline service-api/docker/seeding/start.sh:60]: If new indexes are needed then they will need adding to the terraform for the AWS environments too.

**Review by cooperaj — CHANGES_REQUESTED** [review]

File: service-admin/internal/server/handlers/handler.go [hunk 2]
```diff
@@ -39,7 +39,7 @@ func GetTemplate(ctx context.Context, name string) (*template.Template, error) {
 
 	tmpl, err := t.Get(name)
 	if err != nil {
-		err := fmt.Errorf("%w, template \"%s\" not in available templates", err, name)
+		err := fmt.Errorf("%w, template \"%s\" not in available templates", ErrTemplateNotFound, name)
```
[the remark this answers is not in the document]
Comment (lukejolliffe) [reply service-admin/internal/server/handlers/handler.go:42]: It doesn't like it,  pretty sure it was complaining about this. 

linter err113: do not define dynamic errors, use wrapped static errors instead

File: service-api/docker/seeding/start.sh (hunk 1 above)
Comment (lukejolliffe) [reply service-api/docker/seeding/start.sh:60]: Already there just noticed I missed it from the seeding data

File: service-admin/internal/server/handlers/handler.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (lukejolliffe) [reply service-admin/internal/server/handlers/handler.go]: ^^ don't know why it was appearing, I've taken it back to original functionality and now the linter isn't complaining? So I won't either!

File: service-admin/web/templates/result_code.partial.gohtml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+{{ define "result_code" }}
+  <dl class="govuk-summary-list">
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">
+        Used
+      </dt>
+      <dd class="govuk-summary-list__value">
+        {{ if .Used }}
```
[the remark this answers is not in the document]
Comment (cooperaj) [reply service-admin/web/templates/result_code.partial.gohtml]: This same issue is repeated through out the file :P

**Review by cooperaj — APPROVED** [review]

**Pull request merged**